### ISSUE TITLE
ARIA roles must be contained by parents

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#614]
The Log In link in the footer menu of the Fulcrum app does not have parent ARIA roles present.

Required ARIA parents role not present: menu menubar
.`js.wf-droidserif-n4-active.wf-droidserif-n7-active.wf-droidserif-i7-active.wf-freightsanspro-i4-active.wf-freightsanspro-n3-active.wf-freightsanspro-n6-active.wf-freightsanspro-i6-active.wf-freightsanspro-n4-active.wf-droidserif-i4-active.wf-active > .northwestern > footer > .container-fluid > .row.col-wrap > .col-sm-4.platform.col > .list-unstyled > li:nth-of-type(5) > .login`
`<a class="login" role="menuitem" href="/users/sign_in">Log In</a>`
